usr/lib/linuxmint/mintwelcome/mintwelcome.py

```diff
@@ -1,23 +1,33 @@
 #!/usr/bin/python3
-import apt
-import gettext
-import gi
-import os
-import platform
-import subprocess
-import locale
-gi.require_version("Gtk", "3.0")
-from gi.repository import Gtk, Gio, Gdk, GdkPixbuf
+from os import path as os_path, getenv as os_get_env, system as os_system
+from subprocess import call as subprocess_call, check_output as subprocess_check_output, Popen as subprocess_Popen
```

Comment:
I did that for 2 reasons:
- Importing a specific module, rather than a collection of modules is (usually) good practice
- The prefix is useful to remember the "parent module" (I'm not sure if this is the correct terminology)

Should I import the parent module (original), or remove the prefix (no `as` rename)?

Reply:
> This is meant to discourage using something like `from foo import *`, so you don't up with unexpected global namespace collisions. It's _not_ intended to be used the way you're using it.
> 
> There's no reason to use 'os_get_env' over 'os.getenv', or anything in the os module, for that matter. I'll do it if I'm using just a single function from `ridiculously_long_module_name`, or if I'm repeatedly using `math.pi`, it looks cleaner to maybe `from math import pi as PI`.
> 
> You should just import modules and use them normally for the most part.

Fair enough 👍. Since readability is more important than performance, and the Py runtime is very likely to optimize it, and `import`ing 1 parent module pollutes less the global namespace than multiple modules, then it's safe to conclude that it's better not to use my "prefix approach".

> A bit of info: https://google.github.io/styleguide/pyguide.html#22-imports https://peps.python.org/pep-0008/#imports

Thanks for the links! I'll read them later. I'm busy IRL